### DISCUSSION TITLE
Feature request: default parameters values

### DISCUSSION THREAD

**Discussion by mtlukaszczyk** [spatie/laravel-route-attributes#74] [Ideas]
Hello,

Laravel has default values function for parameters in routing:

```
Route::get('test/test', function($id) {
    echo $id;
})->defaults('id', 1); // or setDefaults(['id' => 1])
```

check `Illuminate\Routing\Route::defaults` or `setDefaults`

This feature is missing in this package.

**Comment by freekmurze** [comment]
Feel free to PR this, make sure to include tests 👍
